lighthouse-core/scripts/lantern/collect/urls.json

```diff
@@ -0,0 +1,101 @@
+[
```

Comment:
might be good to switch to a `.js` file so this can include a comment on how these urls were selected and info on how we might pick a new set in the future